server/routes/treeadoptions/treeadoptions-queries.js

```diff
@@ -18,13 +18,18 @@ export async function adoptTree(adoptedTreeData) {
   const adoptedTreeDataInSnakeCase =
     convertObjectKeysToSnakeCase(adoptedTreeData);
 
-  const query = `INSERT INTO treeadoption (id, id_tree, common, nickname, email)
-    SELECT td.id, td.id_tree, td.common, u.nickname, u.email
+  const query = `INSERT INTO treelikes (id, id_tree, common, nickname, email)
```

Comment:
Is this supposed to insert into the treelikes table? Should it not be the treeadoptions table? Asking since this is the treeadoptions-queries.js file.